src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/CompositionManager.cs

```diff
@@ -272,6 +295,44 @@ public void AddCameraPose(Vector3 cameraPosition, Quaternion cameraRotation, flo
         }
 
 #if UNITY_EDITOR
+        /// <summary>
+        /// Gets the width of the frame for video recording purposes, based
+        /// on the current rendering mode.
+        /// </summary>
+        public int VideoRecordingFrameWidth
+        {
+            get
+            {
+                if (VideoRecordingLayout == VideoRecordingFrameLayout.Composite)
+                {
+                    return UnityCompositorInterface.GetFrameWidth();
+                }
+                else
+                {
+                    return 2 * UnityCompositorInterface.GetFrameWidth();
+                }

```

Comment:
nit: Should GetFrameWidth, GetFrameHeight get handed the VideoRecordingLayout and determine this?

Reply:
I think maybe the name "FrameWidth" is too ambiguous. FrameWidth and FrameHeight refer to the size of the input and output frames from the capture card, which can now be different than the frame size of the recorded video. I could expose a new native method that can get the video recording frame size given a layout though.